### DISCUSSION TITLE
How to check how many servers evobot is in????????????/

### DISCUSSION THREAD

**Discussion by ghost** [eritislami/evobot#1062] [Q&A]
**Is your feature request related to a problem? Please describe.**
A description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A description of what you want to happen.

**Describe alternatives you've considered**
A description of any alternative solutions or features you've considered.

**Additional context**

**Comment by Shaxpartan** [comment]
@ReyanshGupta17  hey um you can check it when you are at adding bot page  [Click Here For Screen Shot](https://cdn.discordapp.com/attachments/918337694118072330/927242520646254592/Screenshot_540.png)

**Comment by Takiyo0** [comment]
`client.guilds.cache.size` to see guilds your bot in.
